ddtrace/tracer.py

```diff
@@ -328,6 +336,19 @@ def wrap_decorator(f):
             else:
                 @functools.wraps(f)
                 def func_wrapper(*args, **kwargs):
+                    # if a wrap executor has been configured, it is used instead
+                    # of the default tracing function
+                    if getattr(self, '_wrap_executor', None):
+                        return self._wrap_executor(
+                            self,
+                            f, args, kwargs,
+                            span_name,
+                            service=service,
+                            resource=resource,
+                            span_type=span_type,
```

Comment:
In the default signature for wrap_executor, are `service, resource, span_type` kwargs or args? This usage, and the one in your test are inconsistent.